crypto/evp/keymgmt_lib.c

```diff
@@ -384,11 +384,6 @@ int evp_keymgmt_util_copy(EVP_PKEY *to, EVP_PKEY *from, int selection)
         return 0;
     }
 
-    if (to->keymgmt == NULL
```

Comment:
Why is this completely deleted??

Reply:
We've already de-referenced `to->keymgmt` in all paths leading to here, it cannot be NULL.

Reply:
@levitte probably should review this?

Reply:
I'll have a look.

Reply:
Actually, it's quite unclear what happens, and what *should* happen when `to->keymgmt == NULL`...  This "quick fix" does not resolve that question.

Reply:
I find it interesting, though, that coverity didn't catch the completely unused `alloc_keydata`.

Reply:
> Actually, it's quite unclear what happens, and what _should_ happen when `to->keymgmt == NULL`... This "quick fix" does not resolve that question.

I know what happens: you get a core dump earlier in the function :)
As for what _should_ happen, I've no idea.

Reply:
That being said, coverity hasn't actually determined that `to->keymgmt` can't be NULL at this point, it only *assumes* this because we've dereferenced it all over the place.  For some reason, it didn't report that we might dereference NULL in one of those code branches.

Reply:
See #11668

Reply:
I'll ditch this change then.

Reply:
It's gone.   Now to figure out which other amendment broke the CIs :(